### PR TITLE
QA fixes for merge 57

### PR DESCRIPTION
**Pull request by thorn573**
Tested on Chrome and Firefox successfully.

Removed 'insert break point' button since it is broken. Will fix in a future release. 

Added direction based conditionals to make alerts more clear. 

After preferences selected, 'add a point' will re-enable. 

Fixed minor errors which occurred when 'delete all' in 'view data' was clicked.

### CODE REVIEW

File: leaflet-treering.js [hunk 1]
```diff
@@ -478,15 +479,11 @@ function MeasurementData (dataObject, Lt) {
           year_adjusted = this.points[i].year;
         };
 
-      } else if (this.points[i - 1].start) { // case 2: previous point is start
-        if (direction == forwardInTime) {
-          year_adjusted = this.points[i + 1].year;
-        } else if (direction == backwardInTime) {
+      } else if (this.points[i - 1].start || this.points[i].start) { // case 2: previous point is start
```
Comment (cmcfadden) [inline leaflet-treering.js:482]: the comment no longer seems to apply to this line. Is this actually fixing what you're trying to fix?

**Comment by cmcfadden** [comment]
another comment I missed earlier - function directionCheck () { should return your constants too, not strings.

File: leaflet-treering.js (hunk 1 above)
Comment (thorn573) [reply leaflet-treering.js:482]: The issue was which year the inserted point selected. My previous code was wrong and could be simplified to a single expression. The comment probably should be modified to 'previous or closest'.

**Comment by cmcfadden** [comment]
looks good!

**Pull request merged**